reconfiguration/include/reconfiguration/dispatcher.hpp

```diff
@@ -38,8 +38,12 @@ class Dispatcher {
   concord::messages::ReconfigurationResponse dispatch(const concord::messages::ReconfigurationRequest&,
                                                       uint64_t sequence_num);
 
-  void addReconfigurationHandler(std::shared_ptr<IReconfigurationHandler> h) { reconfig_handlers_.push_back(h); }
-  void addPruningHandler(std::shared_ptr<IPruningHandler> h) { pruning_handlers_.push_back(h); }
+  void addReconfigurationHandler(std::shared_ptr<IReconfigurationHandler> h) {
```

Comment:
why check for nullptr? in this case it will be silently ignored.

Reply:
if we push nullptr to the handler list and then trying to invoke them, wouldn't this cause a problem?

Reply:
It would, but it's a bug, therefore the earlier we catch it the better